egs/fisher_swbd/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
```

Comment:
Changes are committed accordingly. I add callhm subset for eval2000, and 
fsh subset for rt03.
Please comment.

在 2016/4/14 11:02, Vijayaditya Peddinti 写道:
>
> In egs/fisher_swbd/s5/local/nnet3/run_ivector_common.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59657898>:
>
> > @@ -0,0 +1,141 @@
> > +#!/bin/bash
> > +
> > +. ./cmd.sh
> > +set -e
> > +stage=1
> > +train_stage=-10
> > +generate_alignments=true # false if doing ctc training
>
> tree building has a low sample complexity. It might not be necessary 
> to build the tree using 2100*3 hours of data. So it is definitely 
> overkill to just generate 6300 hrs of alignments just to build a tree. 
> It is sufficient to use the non-sp aligments for the tree building stage.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59657898>
>